/common/httping, /osx/ping: update page and add page (#2773)
tldr-pages/tldr: pages/osx/ping.md

# ping

> Send ICMP ECHO_REQUEST packets to network hosts.

> - Ping `host`:

Reviewer: The backticks aren't necessary here afaik, it's only really for command syntax highlighting.

Perhaps the following would sound better?
```suggestion
- Ping the specified host:
```

`ping {{host}}`

> - Ping a `host` only a specific number of `count`:

Reviewer: ```suggestion
- Ping a host a specific number of times:
```

`ping -c {{count}} {{host}}`

- Ping `host`, specifying the interval in `seconds` between requests (default is 1 second):

`ping -i {{seconds}} {{host}}`

- Ping `host` without trying to lookup symbolic names for addresses:

`ping -n {{host}}`

- Ping `host` and ring the bell when a packet is received (if your terminal supports it):

`ping -a {{host}}`

> - Ping `host` and prints the time a packet was received.  This option is an Apple addition.:

Reviewer: ```suggestion
- Ping `host` and prints the time a packet was received (this option is an Apple addition):
```
